docs/kb/semgrep-supply-chain/ssc-python-lockfiles.md

```diff
@@ -13,7 +13,8 @@ tags:
 
 To correctly scan all dependencies in a project, Semgrep Supply Chain requires a Python lockfile. This article describes methods to generate the following Python lockfiles:
 
-* `requirements.txt`
+* `requirements.txt`, including those in a requirements folder, such as `**/requirements/*.txt`
+* `requirements.pip`
```

Comment:
We also support the requirements without the `s`, so:

- `requirement.txt`, `**/requirement/*.txt`
- `requirement.pip`